airbyte-ci/connectors/pipelines/pipelines/cli/auto_update.py

```diff
@@ -98,36 +98,40 @@ def check_for_upgrade(
     upgrade_error_message = f"""
     🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨
 
-    This version of `airbyte-ci` does not match that of your local airbyte repository.
+    The installed version of `airbyte-ci` does not match the version in your local airbyte repository.
+    This likely means that your installed version of `airbyte-ci` is out of date.
 
     Installed Version: {__installed_version__}.
     Local Repository Version: {latest_version}
 
-    Please upgrade your local airbyte repository to the latest version using the following command:
-    $ {upgrade_command}
-
-    Alternatively you can skip this with the `--disable-update-check` flag.
```

Comment:
the first doesn't seem necessary even before this pr, as we help them upgrade automatically now. the latter can be removed if we do the disabling for them as suggested in this PR. 

If we make it easy to bypass via the prompt, I think they're actually more likely to continue seeing the prompt (and therefore how out of date they are), than if appending `--disable-update-check` is the only way to successfully get the commands running on an old version. Then the flag becomes habit and some people are never prompted again